### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by rugk**
Closes https://github.com/mikifus/padland/issues/5

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <string name="pref_header_general">Einstellungen</string>
+    <string name="title_activity_settings">Einstellungen</string>
+    <string name="action_settings">Einstellungen</string>
+    <string name="go_pad">Erstellen oder anzeigen</string>
+    <string name="title_activity_pad_view">Padland-Anzeige</string>
+    <string name="write_pad_name">Pad-Name</string>
+    <string name="action_newpad">Dokument hinzufügen</string>
+    <string name="action_go_to_list">Zurück zur Pad-Liste</string>
+    <string name="title_activity_new_pad">Pad erstellen</string>
+    <string name="new_pad_explanation">Gib den Pad-Name ein. Wenn es nicht existiert, wird es erstellt; andernfalls wird es dich zum Pad bringen.</string>
```
Comment (rugk) [inline app/src/main/res/values-de/strings.xml]: BTW: Should the  [original string](https://github.com/rugk/padland/blob/patch-1/app/src/main/res/values/strings.xml) really mean "Introduce the pad name"? I think "Enter the pad name" is meant here, is not it?

This is the way I translated it.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    <string name="choose_a_server">Wähle einen Server (optional):</string>
+    <string name="default_pad_server">Standard-Pad-Server</string>
+    <string name="choose_default_pad_server">Wähle einen Standard-Pad-Server für neu erstellte Dokumente.</string>
+    <string name="about_support_text">
+        <p>Diese App ist in der Entwicklung. Um Feedback zu geben und Bugs zu reporten, kontaktiere den Entwickler unter mikifus@gmail.com.</p>\n
+        \n
+        <p>Wenn du zu dem Projekt beitragen willst: Hier ist das Repository:\n
+            <a href='https://github.com/mikifus/padland'>https://github.com/mikifus/padland</a></p>\n
+        \n
+        <p>Deutsche Übersetzung von rugk: <a href='https://github.com/rugk'>https://github.com/rugk</a></p>\n
```
Comment (rugk) [inline app/src/main/res/values-de/strings.xml]: I hope adding myself as a translator is okay. :smiley:

Comment (mikifus) [inline app/src/main/res/values-de/strings.xml]: Indeed!

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    </string>
+    <string name="padinfo_view_pad_button">Zeige Pad</string>
+    <string name="title_activity_pad_info">Dokument-Informationen</string>
+    <string name="title_activity_pad_land_data">PadLand-DataActivity</string>
```
Comment (rugk) [inline app/src/main/res/values-de/strings.xml]: Is this intended to be translated? (basically I just added a hyphen `-` here)

Comment (mikifus) [inline app/src/main/res/values-de/strings.xml]: It is required for the manifest file, but not displayed to the user.

**Pull request merged**